azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/service/sdk/CallingSDKWrapper.kt

```diff
@@ -199,7 +218,7 @@ internal class CallingSDKWrapper(
     }
 
     private fun lobbyNullCheck(future: CompletableFuture<CallCompositeLobbyErrorCode?>): Boolean {
-        if (nullableCall == null || nullableCall?.lobby == null) {
+        if (nullableCall == null || nullableCall?.callLobby == null) {
             future.complete(CallCompositeLobbyErrorCode.UNKNOWN_ERROR)
```

Comment:
Seems a lot of UNKNOWN_ERROR, around this code, is it possible to refine these in the future? UNKNOWN_ERROR provides very little feedback to developers, support, users etc. I.e.
CallCompositeLobbyErrorCode.CALL_OR_LOBBY_NOT_AVAILABLE.

May not be directly related to this.